src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -3065,7 +3081,8 @@ public async Task QueryMethodinfoSelectNotMapped()
         var q = from i in new int[0]
                 where true
                 $$select i;
-");
+",
+            MainDescription($"interface System.Collections.Generic.IEnumerable<out T>"));
```

Comment:
Is this change in behavior OK or not? Previously, there was no quick info on the "select" token. Now it would show the IEnumerable type.

If the new behavior is undesired, I can make a more targeted fix (closer to quick info implementation) to specifically check that the token is a null or a default literal and only then use ConvertedType as a fallback.

Reply:
➡️ The behavior was restored in f06bd472d4988eda30f580614ddbabcc3b212644